### ISSUE TITLE
OEM profile keycaps with dysub printing

### ISSUE THREAD

**Issue by OleUrgast** [hekkelek/amiga-keyboard#1]
Not a real issue, more an idea for the keycaps:
On https://youtu.be/-scV7LLvdh8 you find a video how to print yourself onto keycaps with dysub. As you already put so much work into your project, you might also print to the keycaps. As most printshops already have a printer to print on dysub transfer paper, you need only to make that simple press.
Also there is a keycap designer for cherry keycaps on thingiverse. https://www.thingiverse.com/thing:2783650
The ISO enter generation does not work correctly, but all other keys work fine, even the 1.5 R5 OEM (function keys). ISO Enter in OEM should be easy to get elsethere. So with a resin printer you should be able to make all keycaps yourself.

**Comment by hekkelek** [comment]
Thank you for the suggestion. When I'll have time, I'll definitely try the dye sublimation method. As for the keycaps, I've already printed a 9-unit spacebar using the linked library. I'll upload it sometime.

**Comment by vejinnovem** [comment]
I already designed all keys for Amiga (using KeyV2) with proper sizes (the enormous space as well) but I need to print all of them first before I share them here (and I'll do it with pleasure). I already found some wrong elements in design, so I need to check this printed first.

**Comment by vejinnovem** [comment]
@hekkelek Do you own any 3d model of the bracket to mount the PCB? I don't own any stock one, so I have to put PCB loosely on the case. On the pictures it looks like you have some custom bracket. I write it here since it's also 3d-themed issue.
I'm printing both resin and filament versions of keycaps, so expect something more soon.

**Comment by hekkelek** [comment]
@vejinnovem It's not a 3D printed bracket, but a plate. I've milled it from a 1.6 mm thick PCB using a CNC mill. You can find the design in DXP and Gerber formats in /hw/fab/ as keyboard_full_plate_Rev_B.dxf and keyboard_full_plate_Rev_B.GM4, respectively.

**Comment by vejinnovem** [comment]
@hekkelek you think you could somehow convert it to a 3d object or at least svg?

**Comment by hekkelek** [comment]
@vejinnovem It can not be converted into a 3d objects, because it's a 2D drawing. DXF and Gerber are already CAD software compatible files. Use a tool that can handle them.
I've added an SVG render, but I discourage to use it, because in general, SVGs do not contain information about the size.

**Comment by vejinnovem** [comment]
don't worry, I'll scale it to proper size using calipers and measuring the distance between the holes in real PCB :) Thanks a lot!
By the way, the keys are almost ready:
![IMG_1914](https://user-images.githubusercontent.com/30325517/224557874-8636e16e-9a66-4f5b-812b-3ce7b6b85ce9.JPG)
Just need to fix few of the models to have the full set and I'll be ready to share it!

**Comment by vejinnovem** [comment]
@hekkelek Are you able to tell me what are the dimensions of the PCB I asked you for SVG of?

**Comment by hekkelek** [comment]
@vejinnovem Yes, the outer dimensions of the plate are 459.581 mm x 141.089 mm (18093.75 mil x 5554.688 mil).

**Comment by vejinnovem** [comment]
https://www.thingiverse.com/thing:6067745
there it is, you can add it to your project if you want. Feel free to comment it, I'm not sure If I did anything good. I printed parts of the SVG but somehow the keyboard seems to sit too high. I have to figure it out.
